flake8_trio.py

```diff
@@ -37,7 +37,7 @@ def make_error(error: str, lineno: int, col: int, *args: Any, **kwargs: Any) ->
 
 
 class Flake8TrioVisitor(ast.NodeVisitor):
-    def __init__(self) -> None:
```

Comment:
I was originally also running my code with `mypy --strict` - which is much stricter than pyright and also enforces explicit return values. But I transitioned to only running the code through pyright and especially usage of ` -> None` became inconsistent, so I thought it best to just clean up all the old instances of it